theconfluenceBOT/new.py

```diff
@@ -114,6 +119,8 @@ def departures():
         print("Logged in as Zokalyx!")
     except Exception as e:
         print(e)
+        # Prevents the use of an undefined reddit object
+        sys.exit("Zokalyx login in departures failed.")
```

Comment:
Wouldn't this prevent the bot from replying with "there was a problem"?

Reply:
new commit raises an Exception